File: app/src/main/java/com/kakyiretechnologies/viewbinding/java/BlankFragment.java
package com.kakyiretechnologies.viewbinding.java;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.kakyiretechnologies.viewbinding.R;
import com.kakyiretechnologies.viewbinding.databinding.FragmentBlankBinding;

public class BlankFragment extends Fragment {

    private FragmentBlankBinding binding;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        binding = FragmentBlankBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }


}

File: app/src/main/java/com/kakyiretechnologies/viewbinding/kotlin/BlankFragment.kt
package com.kakyiretechnologies.viewbinding.kotlin

import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import com.kakyiretechnologies.viewbinding.R
import com.kakyiretechnologies.viewbinding.databinding.FragmentBlankBinding

class BlankFragment : Fragment(R.layout.fragment_blank) {

    private lateinit var binding: FragmentBlankBinding
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        binding = FragmentBlankBinding.bind(view)
    }
}